custom_components/media_player_template/media_player.py

```diff
@@ -109,9 +110,10 @@
         vol.Optional(CONF_ENTITY_PICTURE_TEMPLATE): cv.template,
         vol.Optional(CONF_AVAILABILITY_TEMPLATE): cv.template,
         vol.Optional(CURRENT_SOURCE_TEMPLATE): cv.template,
-        vol.Optional(ON_ACTION): cv.SCRIPT_SCHEMA,
-        vol.Optional(OFF_ACTION): cv.SCRIPT_SCHEMA,
+        vol.Required(ON_ACTION): cv.SCRIPT_SCHEMA,
+        vol.Required(OFF_ACTION): cv.SCRIPT_SCHEMA,
```

Comment:
Why did you make them required?

Reply:
I'll be 100% honest. Python is not something I code regularly.
I mainly needed play_pause to be able to use mini_media_player card.

I simply added the same entries as play_action but adapted it for play_pause!

Reply:
Yeah the changes are okay but you've change the ON_ACTION/OFF_ACTION to required parameters

Reply:
I did not modify those lines AFAIK. I took the code on your page and added play_pause in 8 locations.